dts/common/nordic/nrf7120_enga.dtsi

```diff
@@ -844,4 +844,23 @@
 			#mbox-cells = <1>;
 		};
 	};
+
+	wifi: wifi {
+		compatible = "nordic,wifi";
+		status = "disabled";
+
+		wlan0: wlan0 {
+			compatible = "nordic,wlan";
+		};
+
+		wifi-max-tx-pwr-2g-dsss = <21>;
+		wifi-max-tx-pwr-2g-mcs0 = <16>;
+		wifi-max-tx-pwr-2g-mcs7 = <16>;
+		wifi-max-tx-pwr-5g-low-mcs0 = <13>;
+		wifi-max-tx-pwr-5g-low-mcs7 = <13>;
+		wifi-max-tx-pwr-5g-mid-mcs0 = <13>;
+		wifi-max-tx-pwr-5g-mid-mcs7 = <13>;
+		wifi-max-tx-pwr-5g-high-mcs0 = <12>;
+		wifi-max-tx-pwr-5g-high-mcs7 = <12>;
+	};
```

Comment:
Better to have separate dtsi files for 2G and 5G max tx power values so they can be appropriately included as per the build target.

Reply:
do we expect 2G only variant? If not, then we can leave them in a single file.

Reply:
As discussed offline closing this, we can revisit if we have 2G only plans.